src/pages/dapp/index.tsx

```diff
@@ -227,20 +239,28 @@ export function StarRating(props) {
       onChange(rating);
     }
   }, [rating]);
+
+  const handleChangeRating = (ratings: number) => {
+    setHover(0);
+    setRating(ratings);
+  };
+
   return (
     <Row
       className="gap-x-[4px]"
-      onMouseLeave={editable ? () => setRating(rating) : undefined}
+      onMouseLeave={editable ? () => handleChangeRating(rating) : undefined}
     >
       {[...Array(5)].map((_, idx) => {
         idx += 1;
         return (
           <svg
-            onClick={editable ? () => setRating(idx) : undefined}
+            onClick={editable ? () => handleChangeRating(idx) : undefined}
             onMouseEnter={editable ? () => setHover(idx) : undefined}
-            onMouseLeave={editable ? () => setRating(rating) : undefined}
-            className={`icon ${
-              idx <= (rating || hover) ? "icon-filled" : null
+            onMouseLeave={
+              editable ? () => handleChangeRating(rating) : undefined
+            }
+            className={`icon cursor-pointer ${
+              idx <= (hover || rating) ? "icon-filled" : null
```

Comment:
How're we importing classes which are dynamically used? Because it can be missed at the build time.

Reply:
this works fine. this page is not generated statically at build time, and defaults to null anyways. tailwind supports and handles this kind of dynamic styles well

Reply:
it will not tested already